uniparc-rest/src/test/java/org/uniprot/api/uniparc/controller/UniParcDatabaseControllerIT.java

```diff
@@ -0,0 +1,379 @@
+            return GetIdParameter.builder()
+                    .id("UPI0000083A09")
+                    .resultMatcher(jsonPath("$.url", not(emptyOrNullString())))
+                    .resultMatcher(jsonPath("$.messages.*", contains("Resource not found")))
+                    .build();
+        }
+
+        @Override
+        public GetIdParameter withFilterFieldsParameter() {
+            return GetIdParameter.builder()
+                    .id(UNIPARC_ID)
+                    .fields("protein,organism")
+                    .resultMatcher(jsonPath("$.results.*.organism").exists())
+                    .resultMatcher(jsonPath("$.results.*.proteinName").exists())
+                    .resultMatcher(jsonPath("$.results.*.version").doesNotExist())
+                    .build();
+        }
+
+        @Override
+        public GetIdParameter withInvalidFilterParameter() {
+            return GetIdParameter.builder()
+                    .id(UNIPARC_ID)
+                    .fields("invalid")
+                    .resultMatcher(jsonPath("$.url", not(emptyOrNullString())))
+                    .resultMatcher(
+                            jsonPath(
+                                    "$.messages.*",
+                                    contains("Invalid fields parameter value 'invalid'")))
+                    .build();
+        }
+    }
+
+    static class UniParcGetIdContentTypeParamResolver
+            extends AbstractGetIdContentTypeParamResolver {
+
+        @Override
+        public GetIdContentTypeParam idSuccessContentTypesParam() {
+            return GetIdContentTypeParam.builder()
+                    .id(UNIPARC_ID)
+                    .contentTypeParam(
+                            ContentTypeParam.builder()
+                                    .contentType(MediaType.APPLICATION_JSON)
+                                    .resultMatcher(jsonPath("$.results.size()", is(5)))
+                                    .resultMatcher(jsonPath("$.results.*.database").exists())
+                                    .resultMatcher(jsonPath("$.results.*.id").exists())
+                                    .build())
+                    .contentTypeParam(
+                            ContentTypeParam.builder()
+                                    .contentType(UniProtMediaType.TSV_MEDIA_TYPE)
+                                    .resultMatcher(
+                                            content()
+                                                    .string(
+                                                            containsString(
+                                                                    "Database\tIdentifier\tVersion\tOrganism\tFirst seen\tLast seen\tActive")))
+                                    .resultMatcher(
+                                            content()
+                                                    .string(
+                                                            containsString(
```

Comment:
Default TSV changes will impact this result

Reply:
Default TSV is correct.